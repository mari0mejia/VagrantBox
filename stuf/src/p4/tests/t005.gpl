string window_title = "Project 4 t005.gpl -- compare to t005.jpg  ***PRESS q TO QUIT***";
rectangle my_rectangle[3];
circle    my_circle[3];
triangle  my_triangle[3];
pixmap    my_pixmap[3];
textbox   my_textbox[3];
