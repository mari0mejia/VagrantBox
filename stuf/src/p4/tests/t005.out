gpl.cpp::main()
  input file(tests/t005.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
circle my_circle[0]
{
    animation_block = NULL,
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(0),
    y = int(0)
}
circle my_circle[1]
{
    animation_block = NULL,
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(0),
    y = int(0)
}
circle my_circle[2]
{
    animation_block = NULL,
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(0),
    y = int(0)
}
pixmap my_pixmap[0]
{
    animation_block = NULL,
    blue = double(0.5),
    drawing_order = int(0),
    filename = string(""),
    green = double(0.5),
    h = int(10),
    proximity = int(4),
    red = double(0.5),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
pixmap my_pixmap[1]
{
    animation_block = NULL,
    blue = double(0.5),
    drawing_order = int(0),
    filename = string(""),
    green = double(0.5),
    h = int(10),
    proximity = int(4),
    red = double(0.5),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
pixmap my_pixmap[2]
{
    animation_block = NULL,
    blue = double(0.5),
    drawing_order = int(0),
    filename = string(""),
    green = double(0.5),
    h = int(10),
    proximity = int(4),
    red = double(0.5),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle my_rectangle[0]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle my_rectangle[1]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle my_rectangle[2]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
textbox my_textbox[0]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(11),
    proximity = int(4),
    red = double(0),
    size = double(0.1),
    spacing = int(10),
    text = string(""),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
textbox my_textbox[1]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(11),
    proximity = int(4),
    red = double(0),
    size = double(0.1),
    spacing = int(10),
    text = string(""),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
textbox my_textbox[2]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(11),
    proximity = int(4),
    red = double(0),
    size = double(0.1),
    spacing = int(10),
    text = string(""),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
triangle my_triangle[0]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(1),
    h = int(10),
    proximity = int(4),
    red = double(0),
    rotation = double(0),
    size = int(10),
    skew = double(1),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
triangle my_triangle[1]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(1),
    h = int(10),
    proximity = int(4),
    red = double(0),
    rotation = double(0),
    size = int(10),
    skew = double(1),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
triangle my_triangle[2]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(1),
    h = int(10),
    proximity = int(4),
    red = double(0),
    rotation = double(0),
    size = int(10),
    skew = double(1),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
string window_title = "Project 4 t005.gpl -- compare to t005.jpg  ***PRESS q TO QUIT***"
gpl.cpp::main() Calling window->initialize().


gpl.cpp::main() Passing control to window->main_loop().

User quit program. Printing the symbol table.
circle my_circle[0]
{
    animation_block = NULL,
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(0),
    y = int(0)
}
circle my_circle[1]
{
    animation_block = NULL,
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(0),
    y = int(0)
}
circle my_circle[2]
{
    animation_block = NULL,
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(0),
    y = int(0)
}
pixmap my_pixmap[0]
{
    animation_block = NULL,
    blue = double(0.5),
    drawing_order = int(0),
    filename = string(""),
    green = double(0.5),
    h = int(50),
    proximity = int(4),
    red = double(0.5),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(100),
    x = int(0),
    y = int(0)
}
pixmap my_pixmap[1]
{
    animation_block = NULL,
    blue = double(0.5),
    drawing_order = int(0),
    filename = string(""),
    green = double(0.5),
    h = int(50),
    proximity = int(4),
    red = double(0.5),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(100),
    x = int(0),
    y = int(0)
}
pixmap my_pixmap[2]
{
    animation_block = NULL,
    blue = double(0.5),
    drawing_order = int(0),
    filename = string(""),
    green = double(0.5),
    h = int(50),
    proximity = int(4),
    red = double(0.5),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(100),
    x = int(0),
    y = int(0)
}
rectangle my_rectangle[0]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle my_rectangle[1]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle my_rectangle[2]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
textbox my_textbox[0]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(11),
    proximity = int(4),
    red = double(0),
    size = double(0.1),
    spacing = int(10),
    text = string(""),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(0),
    x = int(0),
    y = int(0)
}
textbox my_textbox[1]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(11),
    proximity = int(4),
    red = double(0),
    size = double(0.1),
    spacing = int(10),
    text = string(""),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(0),
    x = int(0),
    y = int(0)
}
textbox my_textbox[2]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(11),
    proximity = int(4),
    red = double(0),
    size = double(0.1),
    spacing = int(10),
    text = string(""),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(0),
    x = int(0),
    y = int(0)
}
triangle my_triangle[0]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(1),
    h = int(11),
    proximity = int(4),
    red = double(0),
    rotation = double(0),
    size = int(10),
    skew = double(1),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
triangle my_triangle[1]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(1),
    h = int(11),
    proximity = int(4),
    red = double(0),
    rotation = double(0),
    size = int(10),
    skew = double(1),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
triangle my_triangle[2]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(1),
    h = int(11),
    proximity = int(4),
    red = double(0),
    rotation = double(0),
    size = int(10),
    skew = double(1),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
string window_title = "Project 4 t005.gpl -- compare to t005.jpg  ***PRESS q TO QUIT***"
